CMakeLists.txt

```diff
@@ -11,6 +11,9 @@
 
 cmake_minimum_required ( VERSION 2.8.8 FATAL_ERROR )
 
+# Use MSVS folders to organize projects on windows
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
```

Comment:
Allow the `FOLDER` property to be set on targets to organize them within the MSVS IDE. I have not added any of these properties, but it's good to include this, so that you can start doing so yourself, if you want to.